.circleci/config.yml

```diff
@@ -1213,6 +1213,46 @@ jobs:
           name: "Go mod tidy"
           command: make mod-tidy && git diff --exit-code
 
+  op-service-rethdb-tests:
```

Comment:
bedrock-go-tests should depend on this job b/c the final job is a required check & it's easier to have one required final check rather manually add each new job to github's required checks.